server/queries/teacher/check-quiz-exists.ts

```diff
@@ -0,0 +1,8 @@
+import dbConnection from '../../database/connections';
+
+export default (quizId: string) => dbConnection.query(
+  `
+    SELECT * FROM quizzes WHERE id = $1
```

Comment:
I'm not sure why you need this query, but if it's only for checking if a test exists then you don't need to return everything '*' from the query and the `id` would be enough for this purpose.